### PR TITLE
GitHub commit to Travis URL

### PR DESCRIPTION
**Pull request by jlacomis**
Implements the functionality for #3

### CODE REVIEW

File: src/abm/util.py [hunk 1]
```diff
@@ -69,10 +69,21 @@ def github_commit_to_travis_build_url(repo: str, commit: str) -> str:
...
-    raise NotImplementedError
+    found = False
+    current_build_num = last_build_num
+    # Dumb linear search
+    while not found:
+        build = token.builds(slug=repo_slug, number=current_build_num)[0]
+        if build.commit.sha == commit:
+            found = True
+            found_id = build.id
+        current_build_num = current_build_num - 1
```
Comment (ChrisTimperley) [inline src/abm/util.py]: `-= 1`

File: src/abm/util.py [hunk 2]
```diff
@@ -69,10 +69,21 @@ def github_commit_to_travis_build_url(repo: str, commit: str) -> str:
...
     # I think this means that we have to do a dumb linear search of all of
     # the builds in the repo to find the one that corresponds to this commit.
-    raise NotImplementedError
+    found = False
+    current_build_num = last_build_num
+    # Dumb linear search
+    while not found:
+        build = token.builds(slug=repo_slug, number=current_build_num)[0]
+        if build.commit.sha == commit:
+            found = True
```
Comment (ChrisTimperley) [inline src/abm/util.py]: You could just use `while True` and `break`, but I'll leave that to you.

**Comment by ChrisTimperley** [comment]
Nice.

File: src/abm/util.py (hunk 1 above)
Comment (jlacomis) [reply src/abm/util.py]: "Hi, my name is Jeremy. I've written a program once or twice."

**Comment by jlacomis** [comment]
Confusingly `after_number` returns the builds _before_ a build number. The Ruby implemenation of the Travis API has a `--limit` flag that you can use to get more than 25 builds at a time, but it doesn't seem to work with TravisPy. I can try to see about getting that working with a fork of TravisPy, but for now this seems to be reasonable.

**Pull request merged**